### PR TITLE
Bug 1832186: Bump all dependencies to kube 1.18

### PR DESCRIPTION
**Pull request by alexander-demichev**
Bump all dependencies to kube 1.18

### CODE REVIEW

**Comment by JoelSpeed** [comment]
This needs the Go 1.13 stuff to be merged first before it will pass tests

File: vendor/github.com/openshift/machine-api-operator/pkg/controller/machine/controller.go [hunk 1]
```diff
@@ -397,19 +396,19 @@ func (r *ReconcileMachine) deleteNode(ctx context.Context, name string) error {
...
+	switch t := err.(type) {
+	case *RequeueAfterError:
+		klog.Infof("Actuator returned requeue-after error: %v", err)
+		return reconcile.Result{Requeue: true, RequeueAfter: t.RequeueAfter}, nil
 	}
 	return reconcile.Result{}, err
 }
 
 func isInvalidMachineConfigurationError(err error) bool {
-	var machineError *MachineError
```
Comment (enxebre) [inline vendor/github.com/openshift/machine-api-operator/pkg/controller/machine/controller.go:409]: this seems to be brining old code?

File: go.mod [hunk 2]
```diff
@@ -5,20 +5,19 @@ go 1.13
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/go-logr/logr v0.1.0
-	github.com/onsi/ginkgo v1.12.0
+	github.com/onsi/ginkgo v1.11.0
```
Comment (enxebre) [inline go.mod:8]: why downgrading ginkgo?

Comment (alexander-demichev) [reply go.mod:8]: Bad rebase :D

**Comment by enxebre** [comment]
can you please tag a bz?

**Pull request merged**